bin/archey

```diff
@@ -76,6 +76,7 @@ case "${packager}" in
     packagehandler=$(brew list -1 | wc -l | awk '{print $1 }')
     ;;
   macports)
+    hash port 2>/dev/null || { echo >&2 "This option requires MapPorts, and you don't seem to have it."; exit 1; }
```

Comment:
It's a [double check](https://github.com/ArielCabib/archey-osx/blob/gracefully-exiting-when-macports-not-installed/bin/archey#L6)
If you got here with an error then you should fix the original check and not this part.